CompilerSource/compiler/compile.cpp

```diff
@@ -161,6 +161,10 @@ dllexport int compileEGMf(EnigmaStruct *es, const char* exe_filename, int mode)
   return current_language->compile(GameData(es), exe_filename, mode);
 }
 
+dllexport int compileProto(const buffers::Project *proj, const char* exe_filename, int mode) {
+  return current_language->compile(GameData(*proj), exe_filename, mode);
+}
```

Comment:
ween

Reply:
I want you to teach me at some point how to do this: write my own DLL compiler

Reply:
Ok, I wasnt planning on doing it again. I just didn't see any point in me have that ability as I most likely will not ever use it.

Reply:
If you did not see it before, then Josh possibly changed the permissions or moved groups around or added you to a contributor group. I'm not sure.